e2e/features/submit-appeal-and-make-payment.feature

```diff
@@ -0,0 +1,308 @@
+Feature: Leadership judge record the appellant decision and reasons
+
```

Comment:
@bradsorour i think we need to change the Title of Feature

Reply:
Yep, that doesn't make sense.

Reply:
Fixed.